### PR TITLE
refine readme add release, golang info

### PR DESCRIPTION
**Pull request by niubell**
### What problem does this PR solve?

Issue Number: ref #5687

#### Tests 
 - No code

##### Will it cause performance regression or break compatibility?
No
##### Do you need to update user documentation, design documentation or monitoring documentation?
No
### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Comment by amyangfei** [comment]
@niubell FYI I have added an issue reference in the issue description

File: README.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
 # TiFlow
 
+[![LICENSE](https://img.shields.io/github/license/pingcap/tiflow.svg)](https://github.com/pingcap/tiflow/blob/master/LICENSE)
+![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/pingcap/tiflow)
+![GitHub Release Date](https://img.shields.io/github/release-date/pingcap/tiflow)
+![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/pingcap/tiflow)
 [![Build Status](https://github.com/pingcap/tiflow/actions/workflows/check_and_build.yaml/badge.svg?branch=master)](https://github.com/pingcap/tiflow/actions/workflows/check_and_build.yaml?query=event%3Apush+branch%3Amaster)
 [![codecov](https://codecov.io/gh/pingcap/tiflow/branch/master/graph/badge.svg)](https://codecov.io/gh/pingcap/tiflow)
 [![Coverage Status](https://coveralls.io/repos/github/pingcap/tiflow/badge.svg)](https://coveralls.io/github/pingcap/tiflow)
```
Comment (amyangfei) [inline README.md:9]: Will we keep both these two coverage report, one for unit test and one for integration test

**Review by amyangfei — APPROVED** [review]

**Review by lance6716 — APPROVED** [review]
please address comment of amyangfei

File: README.md (hunk 1 above)
Comment (niubell) [reply README.md:9]: After EE fix problem of `Coverage Status`, we will check the result and determine one, maybe either of the two will be ok.

**Pull request merged**